rootfs-builder/README.md

```diff
@@ -0,0 +1,62 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+The Kata Containers guest OS is created using `rootfs.sh`.
```

Comment:
"The Kata Containers guest OS rootfs is created using the `rootfs.sh` script."

That's a "bit of a mouthful" so I'm wondering if we can have a "Terms" section or similar so that we can just say "rootfs" everwhere after we've defined that this is a "root filesystem based on a particular Linux distro that will be used as the containers guest O/S" in the Terms section?